coalib/parsing/filters/LanguageFilter.py

```diff
@@ -0,0 +1,27 @@
+def language_filter(bears, languages, log_printer):
+    local_bears, global_bears = bears
+    if languages:
+        local_bears = filter_section_bears_by_languages(local_bears, languages)
+        global_bears = filter_section_bears_by_languages(
+            global_bears, languages)
+    return local_bears, global_bears
```

Comment:
I imagine currently that every filter will contain this code, maybe you invoke in the "filter-core" the filter function twice? Once for local and once for global bears? Then people only have to write the filter^^